index.html

```diff
@@ -1,112 +1,110 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Pokemon Survey</title>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
-  </head>
-  <body>
-    <div class="fixed">
-      <h1 id="title">Survey Page by Tamala and Hakizuwera</h1><hr>
-      <p id="description">
-        This is a basic survey that takes into account, your opinions on the Pokemon game series. It will focus mostly on the Kanto region.
-      </p>
-      <div class="transparent">
-      <form >
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Pokemon Survey</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+    integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  <link href="https://fonts.googleapis.com/css?family=Lato:400,700" rel="stylesheet">
+</head>
+
+<body>
+  <div class="fixed">
+    <h1 id="title">Survey Page by Tamala and Hakizuwera</h1>
+    <hr>
+    <p id="description">
+      This is a basic survey that takes into account, your opinions on the Pokemon game series. It will focus mostly on
+      the Kanto region.
+    </p>
+    <div class="transparent">
+      <form>
         <fieldset>
-        <label for="name" id="name-label"> Name:</label>
-        <input id="name" type="text" placeholder="Enter your name." required>
+          <label id="name-label"> Name:</label>
+          <input id="name" placeholder="Enter your name." required>
```

Comment:
Please it will be nice if you include the type attribute because the type attribute specifies the type of <input> element to display.